src/TextField/TextField.js

```diff
@@ -177,6 +182,10 @@ class TextField extends Component {
      */
     inputStyle: PropTypes.object,
     /**
+     * Whether the input value is required.
+     */
+    isRequired: PropTypes.bool,
```

Comment:
Besides, we are already using *short* version like `open` for the `Dialog` instead of `isOpen`.